src/components/Dashboard/TodoList.vue

```diff
@@ -255,27 +247,56 @@
                   <span v-else>PAID</span>
                 </div>
 
-                <!-- change-requested filing -->
+                <!-- draft continuation filing -->
+                <div
+                  v-else-if="EnumUtilities.isStatusDraft(item) && EnumUtilities.isTypeContinuationIn(item)"
+                  class="todo-list-detail pt-1"
+                >
+                  <span
+                    v-if="getNameRequest"
+                    class="list-item__subtitle"
+                  >
+                    Name Request APPROVED - {{ expiresText(getNameRequest) }}
+                  </span>
+                  <span
+                    v-else
+                    class="list-item__subtitle"
+                  >
+                    DRAFT
+                  </span>
+                </div>
+
+                <!-- change-requested continuation filing -->
                 <div
                   v-else-if="EnumUtilities.isStatusChangeRequested(item)"
-                  class="todo-subtitle pt-1"
+                  class="todo-list-detail pt-1"
                 >
-                  <span class="orange--text text--darken-2">CHANGE REQUESTED</span>
+                  <span class="orange--text text--darken-2 font-weight-bold">CHANGE REQUESTED</span>
                   <span class="vert-pipe" />
-                  <span>PAID (filed by {{ item.submitter }} on <DateTooltip :date="item.submittedDate" />)</span>
+                  <span>Submitted by {{ item.submitter }} on <DateTooltip :date="item.submittedDate" />)</span>
+                  <p
+                    v-if="getNameRequest"
+                    class="list-item__subtitle mb-0"
+                  >
+                    Name Request APPROVED - {{ expiresText(getNameRequest) }}
+                  </p>
                 </div>
 
                 <!-- approved continuation filing -->
```

Comment:
This is actually not just for continuation filings. Please update comment.

(We'll worry about line 291 in the future, if/when we have other approved filings.)

Reply:
Updated the comments